templates/signup.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style> 
+            body {
+                padding-top: 20px;
+                text-align: center;
+                background-image: url("/static/background_images/signup-login-bg.png");
+                background-position: top;
+            }
+            hr {
+                width: 50%;
+            }
+            .border {
+                border-style: solid;
+                background-color: aquamarine;
+                display: inline-block;
+                padding-left: 20px;
+                padding-right: 20px;
+                padding-bottom: 20px;
+            }
+        </style>
+    </head>
+    <body>
+        <div class=border>
+            <h2><u>SIGNUP</u></h2>
+            <ul id="flashes" class=flashes>
+                {% with messages = get_flashed_messages() %}
+                {% if messages %}
+                    {% for message in messages %}
+                        <p>{{message}}</p>
+                    {% endfor %}
+                {% endif %}
+                {% endwith %}
+            </ul>
+            <form method="POST" action="/signup">
+                <input type="text" name="email" placeholder="Enter email" maxlength="100"/>
+                <input type="text" name="password" placeholder="Enter password"/>
+                <input type="submit" value="SUBMIT"/>
```

Comment:
I just realized that I goofed up and forgot add zip code within models.py. I'll fix that in a bit. We also need to add a text input to get the zip code.